File: devops-scm-provider/devops-scm-provider-svn/devops-scm-provider-svn-common/src/main/java/com/tencent/devops/scm/provider/svn/common/SvnWebhookEnricher.java
package com.tencent.devops.scm.provider.svn.common;

import com.tencent.devops.scm.api.WebhookEnricher;
import com.tencent.devops.scm.api.exception.UnAuthorizedScmApiException;
import com.tencent.devops.scm.api.pojo.repository.ScmProviderRepository;
import com.tencent.devops.scm.api.pojo.webhook.Webhook;

public class SvnWebhookEnricher implements WebhookEnricher {

    @Override
    public Webhook enrich(ScmProviderRepository repository, Webhook webhook) throws UnAuthorizedScmApiException {
        return webhook;
    }
}

File: devops-scm-provider/devops-scm-provider-svn/devops-scm-provider-svn-common/src/main/kotlin/com/tencent/devops/scm/provider/svn/common/SvnWebhookEnricher.kt
package com.tencent.devops.scm.provider.svn.common

import com.tencent.devops.scm.api.WebhookEnricher
import com.tencent.devops.scm.api.pojo.repository.ScmProviderRepository
import com.tencent.devops.scm.api.pojo.webhook.Webhook

open class SvnWebhookEnricher : WebhookEnricher {

    override fun enrich(repository: ScmProviderRepository, webhook: Webhook) = webhook
}